src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -116,6 +116,7 @@ void HttpInfo::fill(const BSONObj& bo)
   if (this->custom)
   {
     this->payload  = bo.hasField(CSUB_PAYLOAD)? getStringFieldF(bo, CSUB_PAYLOAD) : "";
+    this->includePayload = this->payload == ""? true : false;
```

Comment:
I have realized on this...

As far as I understand, you are setting the value of includePayload based on another field (payload). As far as I understand, it should be an idependent parametner of the subscription, managed the same way any other CSUB_xxxx in the database.

Have a look to CSUB_ONLYCHANGED (another boolean in the subs information). It should be managed in a similar way.

Reply:
Updated